articles/connections/database/custom-db/templates/change-password.md

```diff
@@ -324,27 +345,28 @@ function changePassword(email, newPassword, callback) {
 ### MySQL
 
 ```sql
-function changePassword (email, newPassword, callback) {
-  var connection = mysql.createConnection({
-    host     : 'localhost',
-    user     : 'me',
-    password : 'secret',
-    database : 'mydb'
+function changePassword(email, newPassword, callback) {
+  const mysql = require('mysql');
+  const bcrypt = require('bcrypt');
+
+  const connection = mysql({
+    host: 'localhost',
+    user: 'me',
+    password: 'secret',
+    database: 'mydb'
   });
 
   connection.connect();
 
-  var query = "UPDATE users SET password = ? WHERE email = ? ";
+  const query = 'UPDATE users SET password = ? WHERE email = ?';
```

Comment:
In Heroku review app, code block renders strangely from here on (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/change-password#mysql